rubicon_ml/repository/base.py

```diff
@@ -937,6 +938,9 @@ def _get_tag_metadata_root(
         else:
             entity_metadata_root = get_metadata_root(project_name, experiment_id)
 
+            # We want to slugify the names of Metrics, Features, and Parameters- not Artifacts, Dataframes, or Experiments
+            if entity_type == "Metric" or entity_type == "Feature" or entity_type == "Parameter":
```

Comment:
nice! can we do `if entity_type in ["Metric", "Feature", "Parameter"]:` instead? its a bit cleaner imo once we have more than two and/or clauses